### PR TITLE
docs: copy tutorial using dapr ambient and kind

### PR DESCRIPTION
**Pull request by mcruzdev**
This pull request aims to solve the issue #12.

It is a copy from this [repo](https://github.com/salaboy/dapr-ambient).

### CODE REVIEW

**Comment by salaboy** [comment]
@mcruzdev we need to get this example but without Knative, as it is not needed to get Ambient working. 
Can you help me out with this?

File: docs/tutorial/dapr-ambient-and-knative.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+We will be creating a local KinD cluster where we will install Knative Serving and Dapr.
+
+For this you will need to install the following CLIs:
+
+- [Install `kubectl`](https://kubernetes.io/docs/tasks/tools/)
+- [Install `kind`](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+- [Install `helm`](https://helm.sh/docs/intro/install/) 
+- [Install `docker`](https://docs.docker.com/engine/install/)
+- [Install the Knative Functions `func` CLI](https://knative.dev/docs/functions/install-func/)
```
Comment (salaboy) [inline docs/tutorial/dapr-ambient-and-knative.md:17]: @mcruzdev remove this, and all references to Knative

File: docs/tutorial/dapr-ambient-and-knative.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+Create a local Kubernetes cluster with: 
+
+```
+cat <<EOF | kind create cluster --config=-
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  extraPortMappings:
```
Comment (salaboy) [inline docs/tutorial/dapr-ambient-and-knative.md:28]: @mcruzdev `kind create cluster` should work now.. as we are not going to use Knative

File: docs/tutorial/dapr-ambient-and-knative.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+  - containerPort: 31080 # expose port 31380 of the node to port 80 on the host, later to be use by kourier or contour ingress
+    listenAddress: 127.0.0.1
+    hostPort: 80
+EOF
+```
+
+Let's now install Knative Serving into the cluster:
```
Comment (salaboy) [inline docs/tutorial/dapr-ambient-and-knative.md:35]: @mcruzdev remove this Knative reference

File: docs/tutorial/dapr-ambient-and-knative.md [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+```
+
+Finally, let's deploy three applications that uses the Dapr StateStore and PubSub components. This are normal/regular Kubernetes applications, using Deployments and Services. To make these apps dapr-aware we just need to add some Dapr annotations:
+
+
+```
+
+```
+
+Let's deploy the apps with:
```
Comment (salaboy) [inline docs/tutorial/dapr-ambient-and-knative.md:198]: @mcruzdev These applications can be found here: https://github.com/salaboy/dapr-example-apps 
But you need to replace how they connect to the ambient instance: 
Doing something like this: 
```
var daprClient dapr.Client
var DAPR_HOST = "my-ambient-dapr-ambient.default.svc.cluster.local"
var DAPR_PORT = "50001"

type MyValues struct {
	Values []string
}


	daprClient, err := dapr.NewClientWithAddress(net.JoinHostPort(DAPR_HOST, DAPR_PORT))
	if err != nil {
		panic(err)
	}

```

When you create a Dapr client, you need to point it to your Dapr Ambient instance.

File: docs/tutorial/dapr-ambient-and-knative.md [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+helm repo update
+helm upgrade --install dapr dapr/dapr \
+--version=1.10.4 \
+--namespace dapr-system \
+--create-namespace \
+--wait
+```
+
+Let's now deploy and configure some Dapr apps! 
+
```
Comment (salaboy) [inline docs/tutorial/dapr-ambient-and-knative.md:120]: @mcruzdev here add the instructions to deploy `dapr-ambient` using the helm chart and set the remoteURL to the subscriber service in kubernetes: 

https://github.com/salaboy/dapr-ambient-examles

File: docs/using-dapr-ambient/dapr-ambient.md [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+```
+
+## Installing Dapr Ambient and all applications
+
+Finally, let's install Dapr Ambient and three applications that uses the Dapr StateStore and PubSub components.
+
+Install Dapr Ambient running this:
+
+```sh
+  helm install my-ambient-dapr-ambient ...
```
Comment (mcruzdev) [inline docs/using-dapr-ambient/dapr-ambient.md:130]: @salaboy
We need to host this helm chart, do you know where?

File: docs/using-dapr-ambient/README.md [hunk 7]
```diff
@@ -0,0 +1,184 @@
...
+```sh
+  curl http://localhost:8888
+```
+
+The response should looks like it:
+
+```
+10
+```
+
```
Comment (salaboy) [inline docs/using-dapr-ambient/README.md:181]: @mcruzdev I think we are missing the subscription.. we need to add that to the demo.. it is quite important.. you need to add the subscriber app to the examples too.

**Comment by salaboy** [comment]
I am closing this, as I've pushed the tutorial already.

**Pull request closed without merging**